coffeecard/CoffeeCard.WebApi/appsettings.json

```diff
@@ -27,7 +27,8 @@
     "ApiUrl": "https://invalidurl.test/",
     "ApiKey": "DummyKey",
     "PaymentPointId": "00000000-0000-0000-0000-000000000000",
-    "WebhookUrl": "https://invalidurl.test/"
+    "WebhookUrl": "https://invalidurl.test/",
+    "AnalogAppRedirectUri": "schema-environment://invalid"
```

Comment:
I imagine that it will be useful that for local development this is a valid URL such that you can spin up the backend and app together. I suggest the default value is the dev value